packages/loaders/demo/stories/SkeletonStory.tsx

```diff
@@ -50,7 +50,7 @@ export const SkeletonStory: Story<IArgs> = ({ backgroundColor, count = 1, typesc
   return (
     <div
       style={{
-        backgroundColor: backgroundColor || (args.isLight ? PALETTE.kale[600] : undefined),
+        backgroundColor: backgroundColor || (args.isLight ? PALETTE.kale[800] : undefined),
```

Comment:
Bumping the shade in accordance to the handy-dandy v8-to-v9 color mapping tool. 👍